rfc/text/advanced/ap_09_rpc_progressive_call_results.md

```diff
@@ -332,7 +332,7 @@ The message flow for progressive results cancellation involves:
 
 Note: Any `ERROR` returned by the *Callee*, in response to the `INTERRUPT`, is ignored (same as in call canceling when mode="killnowait"). So, it is not necessary for the *Callee* to send an `ERROR` message.
 
-**Ignoring Progressive Call Requests**
+**Ignoring Progressive Call Results Requests**
```

Comment:
Double plural "Results Requests" doesn't roll off the tongue. May I suggest "Ignoring Requests for Progressive Call Results"?

Reply:
+1